vanilla-framework/vanilla-framework
# Fix media object linting errors

## Done

- Fixed linting errors on media object pattern (space-around-operator). It's only picked up in Jenkins for some reason, so I can't release until this is merged (issue filed #1637) 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/media-object/media-object/
- Check that the media object pattern looks the same
